src/components/Footer/Footer.tsx

```diff
@@ -13,10 +13,10 @@ import SocialList from './SocialList';
 import styles from './Footer.module.css';
 
 const Footer = () => {
-  const [isShowModal, setIsShowModal] = useState<'feedback' | null>(null);
+  const [showFeedbackModal, setShowFeedbackModal] = useState(false);
 
-  const toggleModal = (name: typeof isShowModal) => () => {
-    setIsShowModal((p) => (p ? null : name));
+  const toggleFeedbackModal = () => {
+    setShowFeedbackModal((p) => !p);
```

Comment:
I think you have only one Modal here, you don't need to specify your name, name it like isModalVisible in useState and handleModalVisibility in func